### PR TITLE
Add a section about reproducibility.

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: reproducible_analyses.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+We'd like to be able to reproduce analysis results easily. While that's not
+always fully achievable, the strategies below help significantly. Note that
+this only about analyses run on *production* data, not the development and
```
Comment (pdiakumis) [inline reproducible_analyses.md:3]:
```suggestion
this is only about analyses run on *production* data, not the development and
```

File: reproducible_analyses.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+testing of new code.
+
+1.  *Only run code that has been committed to a repository.*
+
+    If locally changed code is used to produce analyses results, those
```
Comment (pdiakumis) [inline reproducible_analyses.md:8]:
```suggestion
    If locally changed code is used to produce analysis results, those
```

**Review by pdiakumis — APPROVED** [review]
Looks great, thank you!

**Comment by lgruen** [comment]
Thanks for the careful review, Peter!

**Pull request merged**